src/nvim/option_defs.h

```diff
@@ -581,6 +581,7 @@ EXTERN int p_sm;                // 'showmatch'
 EXTERN int p_smd;               // 'showmode'
 EXTERN long p_ss;               // 'sidescroll'
 EXTERN long p_siso;             // 'sidescrolloff'
+EXTERN int p_sdt;               // 'smalldeletethreshold'
```

Comment:
Not sure about this option name, it's unusually long. 'delcombine' is the only existing option from which precedent can be drawn...

Reply:
I'll think about it